src/app/MessageDef/InvokeRequestMessage.cpp

```diff
@@ -152,5 +152,14 @@ CHIP_ERROR InvokeRequestMessage::Builder::EndOfInvokeRequestMessage()
     }
     return GetError();
 }
+
+uint32_t InvokeRequestMessage::Builder::GetSizeToEndInvokeRequestMessage()
+{
+    // This encodes uint8_t into Tag 0xFF. This means 1 bytes for tag, 1 byte for length, 1 byte for value
```

Comment:
```suggestion
    // EncodeInteractionModelRevision() encodes a uint8_t with context tag 0xFF. This means 1 control byte,
    // 1 byte for the tag, 1 byte for the value.
```
There is no "byte for length" involved here....

Please add a comment in EndOfInvokeRequestMessage that if any of that code changes this code needs to be updated.